dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets

```diff
@@ -1435,44 +1245,6 @@
     </PropertyGroup>
   </Target>
 
-  <!-- ================================================== -->
-  <!-- Create platform-specific artifacts for app bundle. -->
-  <!-- ================================================== -->
-
-  <PropertyGroup>
-    <_CreateAppxBundlePlatformSpecificArtifactsDependsOn>
-      $(Before_CreateAppxBundlePlatformSpecificArtifacts)
-
-      _GetPackageProperties;
-
-      _GetDefaultResourceLanguage;
-
-      _CreatePriConfigXmlForSplitting;
```

Comment:
>_CreatePriConfigXmlForSplitting; [](start = 6, length = 32)

are these tasks still called?

Reply:
some may not be, i'll remove them in round 3

Reply:
for those still need to be run, are they still running when you remove this?

Reply:
yup, e.g., _GetDefaultResourceLanguage

Reply:
any task in the list with PRI needs to run, e.g. _ExpandMainPriFile?

Reply:
I suspect that any target in this list that's not referenced by another target in this file can go.